internal/controller/suite_test.go

```diff
@@ -46,9 +46,10 @@ var (
 )
 
 const (
-	defaultChiaImageTag         = "latest"
-	defaultChiaExporterImageTag = "latest"
-	defaultChiaCAImageTag       = "latest"
+	defaultChiaImageTag            = "latest"
+	defaultChiaExporterImageTag    = "latest"
+	defaultChiaHealthcheckImageTag = "main"
```

Comment:
The next time chia-healthcheck releases, this should be updated to use the latest tag here, and in the API struct's default tag. For now, chia-healthcheck only has an image built off main.

Reply:
I tagged a 0.2.1